### PR TITLE
release 0.6.0

### PR DESCRIPTION
**Pull request by Jimexist**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jobs:
           export RUSTFLAGS='-C target-cpu=skylake'
           docker run --rm -v $(pwd):/io \
             --workdir /io \
-            konstin2/maturin:v0.12.16 \
+            konstin2/maturin:latest \
```
Comment (houqp) [inline .github/workflows/build.yml:108]: sure we don't want to pin the version anymore? :D

**Review by houqp — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (Jimexist) [reply .github/workflows/build.yml:108]: i plan to drop support for manylinux soon...

Comment (houqp) [reply .github/workflows/build.yml:108]: Interesting, how do you plan to support binary wheel releases for linux users without manylinux?

Comment (Jimexist) [reply .github/workflows/build.yml:108]: sorry i didn't make it clearly - i meant to replace the existing docker image approach of [`konstin2/maturin`][1] soon. i guess one approach is to just `pip install maturin` within GitHub action

[1]: https://github.com/PyO3/maturin/blob/main/Dockerfile

**Comment by Jimexist** [comment]
new release is at release-0.6.0 https://github.com/datafusion-contrib/datafusion-python/releases/tag/0.6.0

**Pull request merged**